easybuild/framework/easyconfig/types.py

```diff
@@ -433,6 +433,26 @@ def to_checksums(checksums):
 
     return res
 
+def ensure_iterable_license_specs(specs):
+    """
+    Ensures that the provided license file/server specifications are of correct type and converts
+    them to either a list or a tuple.  The input can either be None, a string, or a list/tuple of
+    strings.
+
+    :param specs: License file/server specifications as provided via `license_file` easyconfig parameter
+    """
+    if specs is None:
+        license_specs = [None]
+    elif isinstance(specs, basestring):
+        license_specs = [specs]
+    elif isinstance(specs, (list, tuple)) and all(isinstance(x, basestring) for x in specs):
+        license_specs = specs
```

Comment:
I would ensure a consistent return type here, either a list of a tuple...

To ensure a list, use `list(specs)` here.

To ensure a tuple, use `tuple(specs)` here and `(None,)` and `(specs,)` above.